src/renderer/src/components/settings-modal/audio-apis.tsx

```diff
@@ -8,13 +8,25 @@ export interface AudioApisProps {
 }
 
 const AudioApis: React.FC<AudioApisProps> = ({ apis, selectedApiId, selectApi }) => {
+  let apiValue = -1;
+
+  // Help the user out a bit by defaulting the audio API to the only valid
+  // entry if no API was selected and the list of available APIs only
+  // has one entry.
+  if (apis.some((api) => api.id === selectedApiId)) {
+    apiValue = selectedApiId;
+  } else if (apis.length === 1) {
+    apiValue = apis[0].id;
+    selectApi(apiValue);
```

Comment:
This does automatically set the value in the backend right?

Reply:
Yep, it flows all the way through.